components/PostImage/PostImage.js

```diff
@@ -23,17 +58,22 @@ export function PostImage({
         [styles.isLoading]: isLoading,
       })}
       style={{ aspectRatio, width: `${fraction * 100}%` }}
+      onBlur={() => setIsTagIconVisible(false)}
```

Comment:
This is old?

Reply:
Yes, when I had this PR set as a draft, I was experimenting with onClick for the tagIcon.